_posts/2021-04-16-rustc-ios-and-an-m1.md

```diff
@@ -0,0 +1,178 @@
+---
+permalink: "/{{ year }}/{{ month }}/{{ day }}/rustc-ios-and-an-m1"
+title: "This Week in Glean: rustc, iOS and an M1"
+published_date: "2021-04-16 15:00:00 +0100"
+layout: post.liquid
+data:
+  route: blog
+  tags:
+    - mozilla
+    - rust
+---
+
+(“This Week in Glean” is a series of blog posts that the Glean Team at Mozilla is using to try to communicate better about our work. They could be release notes, documentation, hopes, dreams, or whatever: so long as it is inspired by Glean.)
+All "This Week in Glean" blog posts are listed in the [TWiG index](https://mozilla.github.io/glean/book/appendix/twig.html)
+(and on the [Mozilla Data blog](https://blog.mozilla.org/data/category/glean/)).
+This article is [cross-posted on the Mozilla Data blog]().
+
+---
+
+Back in February I got an M1 MacBook.
+That's Apple's new ARM-based hardware.
+
+I got it with the explicit task to ensure that we are able to develop and build [Glean] on it.
+We maintain a Swift language binding, targeting iOS, and that one is used in [Firefox iOS]
+and also continue supporting development of the [Swift language bindings][swift lb], targeting iOS.
```

Comment:
This whole sentence is a bit confusing, I don't understand what you are trying to say.

Reply:
Oh, yeah, re-reading that it looks like some editing gone wrong, mixing two previous sentences. Fixing!